### ISSUE TITLE
Useless Errors!

### ISSUE THREAD

**Issue by hmoradian** [imanghafoori1/laravel-microscope#43]
Hello MR. Ghafoori,
I have a method with this codes:
```
    public function all()
    {
        return User::where('role_id', 4)->get();
    }
```
Now, After run "php artisan check:all", part of message is following:
```
  |*************************************************************************************|
  |                                                                                     |
  | 11 Class does not exist:                                                            |
  |    App\Repositories\School\4  <==== does not exist.                            |
at app\Repositories\School\TeacherEloquent.php:14                                       |
  |                                                                                     |
```
Package version: **0.1.50**
Line 14   ===>   return User::where('role_id', **4**)->get();
**4** in this line is int **value**, not a **class** !!!

**Comment by imanghafoori1** [comment]
Hhhmmm,  It's strange... 
Will fix it.

**Comment by imanghafoori1** [comment]
fixed on 0.1.52
